composeApp/src/commonMain/kotlin/com/daedan/festabook/data/repository/FestivalNotificationRepositoryImpl.kt

```
@@ -46,29 +53,43 @@ class FestivalNotificationRepositoryImpl(
 
     override suspend fun deleteFestivalNotification(): Result<Unit> {
         val festivalId =
-            festivalLocalDataSource.getFestivalId() ?: return Result.failure(
-                IllegalStateException(),
-            )
+            festivalLocalDataSource.getFestivalId().first() ?: run {
+                // 여기에 로그 달아주세요잉
+                return Result.failure(IllegalStateException())
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)